src/chronometer.js

```diff
@@ -1,30 +1,42 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
     // ... your code goes here
+
+    setInterval(() => {
+      return this.currentTime++;
+    }, 1000);
   }
 
   getMinutes() {
     // ... your code goes here
+    return Math.floor(this.currentTime / 60);
   }
 
   getSeconds() {
     // ... your code goes here
+
+    return this.currentTime % 60;
   }
 
-  computeTwoDigitNumber(value) {
+  computeTwoDigitNumber(number) {
     // ... your code goes here
+    return number > 9 ? "" + number : "0" + number;
   }
 
   stop() {
     // ... your code goes here
+    clearInterval(null);
```

Comment:
You could have saved the ID that the `setInterval` returns that should be saved on the `this.intervalId` and you could use that one to clear the interval